torch/nn/__init__.py

```diff
@@ -3,3 +3,9 @@
 from .parallel import DataParallel
 from . import init
 from . import utils
+
+# Re-export submodules
+
+from torch.nn import intrinsic as intrinsic
+from torch.nn import quantizable as quantizable
+from torch.nn import quantized as quantized
```

Comment:
Should these be `from .nn` to match the above?

Reply:
(I don't know if there's a semantic difference off the top of my head.)

Reply:
From `.nn` seems safer to me but IDK either; @brettcannon, is there a semantic difference?

Reply:
It's essentially stylistic/preference; I personally prefer relative imports as it makes it clearer that the module is from the same package and renames won't affect anything. But it does make relocating the file in the future slightly harder.